test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
```

Comment:
This installs the build dependencies of the "cockpit" package in the repos, which might not be the same as the build dependencies of the sources that are going to be compiled, no?